src/components/button/button.tsx

```diff
@@ -36,6 +36,11 @@ export class Button {
    */
   @Prop() modalTriggerId = '';
 
+  /**
+   *  Accepts the id of the fw-toast component to open it on click
+   */
+  @Prop() toastTriggerId = '';
```

Comment:
Do we need this? Generally, modals have button use cases, toasts generally don't.